charts/s3gw/templates/configmap.yaml

```diff
@@ -8,6 +8,6 @@ metadata:
 {{ include "s3gw.labels" . | indent 4}}
 data:
 {{- if .Values.ui.enabled }}
-  RGW_SERVICE_URL: 'https://{{ .Values.serviceName }}.{{ .Values.publicDomain }}'
+  RGW_SERVICE_URL: '{{ .Values.ui.backendProtocol }}://{{ .Values.serviceName }}.{{ .Values.publicDomain }}'
```

Comment:
There os am appropriate default missing from the `values.yaml`, but more importantly, when we're able to determine whether or not the s3gw even has TLS certificates in the `deployment.yaml` file, why do we need the user to set this here? Can't we just use that condition to switch between `http` and `https` here based on whether or not the `s3gw` container even has TLS certificates or not?

Reply:
:+1: 

backendProtocol would allow a combination of https on frontend and http in FE->RGW communication. 

Without backendProcotol this is simpler though. Hence removed.